cmd/akash/session/base.go

```diff
@@ -4,24 +4,46 @@ import (
 	"os"
 	"path"
 
-	"github.com/ovrclk/akash/cmd/akash/constants"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
+const (
+	flagRootDir = "data"
+	flagNode    = "node"
+	flagNonce   = "nonce"
+	flagKey     = "key"
+	keyDir      = "keys"
+	codec       = "english"
+	flagKeyType = "type"
+	keyType     = "ed25519"
+	flagNoWait  = "no-wait"
+	flagHost    = "host"
+	password    = "0123456789"
```

Comment:
since there's no more `constants.` qualifier, make these `defaultPassword`, `defaultCodec`, etc...

Reply:
we should eventually not have a default password if there's a build flag indicating that a release is being built.